src/city-council-district/city-council-district.service.ts

```diff
@@ -28,6 +32,32 @@ export class CityCouncilDistrictService {
     return await this.cityCouncilDistrictRepository.findTiles(params);
   }
 
+  async findGeoJsonById({
+    cityCouncilDistrictId,
+  }: FindCityCouncilDistrictGeoJsonByCityCouncilDistrictIdPathParams) {
+    const cityCouncilDistrictGeoJson =
+      await this.cityCouncilDistrictRepository.findGeoJsonById({
+        cityCouncilDistrictId,
+      });
+    if (cityCouncilDistrictGeoJson === undefined)
+      throw new ResourceNotFoundException();
+
+    const geometry = JSON.parse(cityCouncilDistrictGeoJson.geometry);
```

Comment:
Any reason you aren't doing `as MultiPolygon` here like we do in the tax lot service? I know it puts a type on the result that we don't really know for sure, but it does seem better than an implicit `any`.

Reply:
I'll add it